Knoema.Client/Data/RegularTimeSeriesRawData.cs

```diff
@@ -10,6 +10,6 @@ public class RegularTimeSeriesRawData : TimeSeriesRawData
 		public string Frequency { get; set; }
 		public string Unit { get; set; }
 		public float Scale { get; set; }
-		public IEnumerable<object> Values { get; set; }
+		public IEnumerable<string> Values { get; set; }
```

Comment:
Are you sure this change will not break backward compatibility?

Reply:
Changed by mistake. Reverted.